### PR TITLE
Bugfix/#351 dropdown list layout

### PR DESCRIPTION
**Pull request by pleszkiewicz**
#351 

Changelist:
- fixed issue with dropdown list layout
- improved dropdown list background
- support for multiSelectMode

dropdown_list

### CODE REVIEW

File: ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift [hunk 1]
```diff
@@ -31,127 +31,91 @@ import SceneKit
...
+        assert(labelNode == nil, "labelNode must not be initialized!")
+        labelNode = LabelNode()
+        labelNode.defaultTextSize = UiDropdownListNode.defaultTextSize
+        contentNode.addChildNode(labelNode)
 
-        gridLayoutNode.addChild(labelNode)
-        gridLayoutNode.addChild(iconNode)
-
-        contentNode.addChildNode(gridLayoutNode)
+        iconNode = NodesFactory.createPlaneNode(width: 1, height: 1, image: SystemIcon("chevron-down").getImage())
```
Comment (SPaluchiewiczSofomo) [inline ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift]: 👍

File: ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift [hunk 2]
```diff
@@ -162,7 +126,11 @@ import SceneKit
 
     @discardableResult
     @objc override func addChild(_ child: TransformNode) -> Bool {
-        guard let dropDownListItem = child as? UiDropdownListItemNode else { return false }
+        guard let dropDownListItem = child as? UiDropdownListItemNode else {
+            return super.addChild(child)
```
Comment (SPaluchiewiczSofomo) [inline ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift]: Is it safe? Looks like this line allows to `inject` other nodes into DropdownList.

File: ios/RNMagicScript/components/Utils/Extensions/UIFont+Lomino.swift [hunk 3]
```diff
@@ -22,11 +22,11 @@ extension UIFont {
         let name: String = UIFont.fontName(from: style, weight: weight)
         guard let font = UIFont(name: name, size: size) else {
             // return system font in case Lomino font not installed
-            let systemFont = UIFont.systemFont(ofSize: size, weight: .regular)
+            let systemFont = UIFont.systemFont(ofSize: size, weight: UIFont.UIFontWeight(from: weight))
```
Comment (SPaluchiewiczSofomo) [inline ios/RNMagicScript/components/Utils/Extensions/UIFont+Lomino.swift]: Nice catch!

File: ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift (hunk 2 above)
Comment (pleszkiewicz) [reply ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift]: This is safe and intended. I think we should allow adding any child-component to any parent-component and leave the decision (whether it makes sense) to the developer 🤔

Comment (SPaluchiewiczSofomo) [reply ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift]: 👍

**Pull request merged**